docs/content/2.deploy/20.providers/cloudflare.md

```diff
@@ -93,41 +91,31 @@ on:
       - main
 
 jobs:
-  ci:
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os: [ ubuntu-latest ]
-        node: [ 14 ]
-
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy
     steps:
-      - uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node }}
-
       - name: Checkout
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
 
-      - name: Cache node_modules
-        uses: actions/cache@v2
+      - name: Setup Node.js 16.x
+        uses: actions/setup-node@v3
         with:
-          path: node_modules
-          key: ${{ matrix.os }}-node-v${{ matrix.node }}-deps-${{ hashFiles(format('{0}{1}', github.workspace, '/yarn.lock')) }}
```

Comment:
I've removed the caching as it looks like an extra step unrelated to this document
(and I feel that keeping things scoped/minimal/simple makes for a better/more useful yml example here)